libtiledbsc/include/tiledbsc/soma_collection.h

```diff
@@ -0,0 +1,45 @@
+#ifndef SOMA_COLLECTION_H
+#define SOMA_COLLECTION_H
+
+#include <tiledb/tiledb>
+#include <tiledb/tiledb_experimental>
+
+namespace tiledbsc {
+using namespace tiledb;
+
+class SOMACollection {
+   public:
+    //===================================================================
+    //= public static
+    //===================================================================
+    static SOMACollection open(std::string_view uri);
+
+    //===================================================================
+    //= public non-static
+    //===================================================================
+    SOMACollection(std::string_view uri);
+
+    std::map<std::string, std::string> list_somas();
```

Comment:
ditto the above -- if this is a map from name to URI, let's say that as a comment

Reply:
Further should this be an `unordered_map`?